.husky/eslint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
I'd add a simple comment explaining what it does. Like "Lints staged files before committing".